ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/NewTabPageViewController.swift

```diff
@@ -468,8 +472,11 @@ class NewTabPageViewController: UIViewController {
     )
   }
 
-  private func showAndFadeInCollectionView() {
+  private func fadeInCollectionViewAndShowVideoButtons() {
+    videoButtonsView.isHidden = true
+    gradientView.isHidden = false
     collectionView.isHidden = false
+
```

Comment:
nit do we need this empty line because `collectionView.isHidden = false` and `collectionView.alpha = 0` are related

Reply:
Fixed